src/pipelines/master_pipeline.py

```diff
@@ -16,14 +19,17 @@ def main(argv):
     controls the over-arching implmentation of the algorithms
     '''
     directory, features, algorithms, parameters = argv
+    
+    # Create a numpy random state
+    random_state = np.random.RandomState(parameters['seed'])
 
     #parsing
     print("parsing json data...",file=sys.stderr)
-    clusters, order, data, test_clusters, test_order, test_data, corpusdict = parse_json.main([directory, parameters])
+    clusters, order, data, test_clusters, test_order, test_data, corpusdict = parse_json.main(directory, parameters)
```

Comment:
It appears that changing the seed values alters the novel/duplicate observation counts. This is likely due to the "if random.random > 0.2" logic in the parse_json function.